test/iron-iconset-svg.html

```diff
@@ -109,39 +127,71 @@
-          var transform = computedStyle.transform || computedStyle.webkitTransform;
-          expect(transform).to.be.eql('matrix(-1, 0, 0, 1, 0, 0)');
-        });
+          setup(function() {
+            iconset = fixture(iconsetFixture);
+            iconContainer = fixture(iconContainerFixture);
+          });
 
-        test('icons not marked as mirror-in-rtl are not mirrored', function() {
-          iconset.applyIcon(rtlContainer, 'circle');
-          var svg = rtlContainer.firstElementChild;
-          var computedStyle = window.getComputedStyle(svg);
-          var transform = computedStyle.transform || computedStyle.webkitTransform;
-          expect(transform).to.be.eql('none');
-        });
+          test('icons marked as mirror-in-rtl are mirrored', function() {
+            iconset.applyIcon(iconContainer, 'rect');
+            var svg = iconContainer.firstElementChild;
+            var computedStyle = window.getComputedStyle(svg);
+            var transform = computedStyle.transform || computedStyle.webkitTransform;
+            expect(transform).to.be.eql('matrix(-1, 0, 0, 1, 0, 0)');
+          });
+
+          test('icons not marked as mirror-in-rtl are not mirrored', function() {
+            iconset.applyIcon(iconContainer, 'circle');
+            var svg = iconContainer.firstElementChild;
+            var computedStyle = window.getComputedStyle(svg);
+            var transform = computedStyle.transform || computedStyle.webkitTransform;
+            expect(transform).to.be.eql('none');
+          });
+
+          test('getComputedStyle is called no more once for all icons', function() {
+            sinon.spy(window, 'getComputedStyle');
+
+            for (var i = 0; i < 3; ++i) {
+              iconset.applyIcon(iconContainer, 'rect');
+            }
+
+            expect(window.getComputedStyle.callCount).to.be.lessThan(2);
+            window.getComputedStyle.restore();
+          });
+        };
+
+        rtlSuite('MirroredIconsetSvg', 'RtlContainer');
+
+        suite('when preferring the global RTL attribute', function() {
 
-        test('many mirrored icons only call getComputedStyle once', function() {
-          sinon.spy(window, 'getComputedStyle');
+          suite('and dir="rtl" on <body>', function() {
+            setup(function() {
+              document.body.setAttribute('dir', 'rtl');
+            });
 
-          for (var i = 0; i < 3; ++i) {
-            iconset.applyIcon(rtlContainer, 'rect');
-          }
+            teardown(function() {
+              document.body.removeAttribute('dir');
+            });
 
-          expect(window.getComputedStyle.callCount).to.be.eql(1);
-          window.getComputedStyle.restore();
+            rtlSuite('GloballyMirroredIconsetSVG', 'DefaultContainer');
+          });
+
+          suite('and dir="rtl" on <html>', function() {
+            setup(function() {
+              document.documentElement.setAttribute('dir', 'rtl');
```

Comment:
maybe call out `html` explicitly? is there a document.html or am i smoking crack?

Reply:
The crack thing. https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement

Reply:
+1 definitely crack